app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
   <body>
     <%= render "layouts/shared/flash_messages" %>
     <% if Current.tenant.present? %>
+      <!-- //? set to org_navbar to review ?// -->
```

Comment:
Would be good to keep the same naming and just overwrite the current "layouts/shared/navbar" with your new navbar logic?

Reply:
Sounds good. I will drop the original navbar and replace with v2.

Reply:
navbar updated.
org-navbar deleted.
application.html.erb note removed
pushed